fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/BindSink.java

```diff
@@ -140,13 +147,28 @@ public List<Rule> buildRules() {
                                                     column.getName()
                                             ));
                                         } else {
-                                            columnToOutput.put(column.getName(),
-                                                    new Alias(Literal.of(column.getDefaultValue())
-                                                            .checkedCastTo(DataType.fromCatalogType(column.getType())),
-                                                            column.getName()));
+                                            if (column.getDefaultValueExpr() == null) {
+                                                columnToOutput.put(column.getName(),
+                                                        new Alias(Literal.of(column.getDefaultValue())
+                                                                .checkedCastTo(
+                                                                        DataType.fromCatalogType(column.getType())),
+                                                                column.getName()));
+                                            } else {
+                                                Expression defualtValueExpression = FunctionBinder.INSTANCE.rewrite(
```

Comment:
add comment to explain which scene getDefaultValueExpr is null, and which is not null